cuml/ridge_regression_demo.ipynb

```diff
@@ -98,10 +96,12 @@
    "outputs": [],
    "source": [
     "%%time\n",
-    "X_cudf = gd.DataFrame.from_pandas(X_train)\n",
-    "X_cudf_test = gd.DataFrame.from_pandas(X_test)\n",
+    "X, y = make_regression(n_samples=n_samples, n_features=n_features, random_state=0)\n",
```

Comment:
different random_state values used for the two datasets

Reply:
Fixed in local.